### PR TITLE
Flask compatible url_for template

### PR DESCRIPTION
**Pull request by iho**

### CODE REVIEW

File: aiohttp_jinja2/__init__.py [hunk 1]
```diff
@@ -16,6 +16,14 @@
 def setup(app, *args, app_key=APP_KEY, **kwargs):
     env = jinja2.Environment(*args, **kwargs)
     app[app_key] = env
+    
+    def url_for(name,  **kwargs):
+        query = kwargs.get('query')
```
Comment (asvetlov) [inline aiohttp_jinja2/__init__.py]: Don't need to extract query explicitly

**Comment by asvetlov** [comment]
Please add tests

File: aiohttp_jinja2/__init__.py (hunk 1 above)
Comment (iho) [inline aiohttp_jinja2/__init__.py]: I test it before commiting and it required.

**Comment by iho** [comment]
@asvetlov I cannot add test.
```
    def test_url_for(self):
        @aiohttp_jinja2.template('tmpl.jinja2')
        @asyncio.coroutine
        def index(request):
            return {}
        @asyncio.coroutine
        def go():
            app = web.Application(loop=self.loop)

            app.router.add_route('GET', '/{name}', index, name='index')

            aiohttp_jinja2.setup(app, loader=jinja2.DictLoader(
                {'tmpl.jinja2': "{{ url_for('index', name='John_Doe')}}"}))

            req = self.make_request(app, 'GET', '/John_Doe')
            text = yield from req.text()
            import ipdb; ipdb.set_trace()

            self.assertIsInstance(env, jinja2.Environment)
            self.assertIs(env, aiohttp_jinja2.get_env(app))

        self.loop.run_until_complete(go())

 ```

**Comment by jettify** [comment]
@iho  You can add test, that required ``url``can be found inside rendered page. Something like:

```python
body = yield from req.text()
self.assertIn('/path/to/John_Doe' , body)
```
or use regular expression to extract ``url`` and then test it against expected. But I think first approach is simpler and better.

**Comment by asvetlov** [comment]
Why `url_for()`? 
We use `ulr()` in routes.

**Comment by iho** [comment]
@asvetlov because it easier when url_for and app already in jinja context.

**Comment by iho** [comment]
What if you want move static to subdomain static.example.com ?

**Comment by asvetlov** [comment]
You motivation for adding new function to jinja2 environment is clean.
I just guess rename it to `url()`.

**Pull request merged**